### ISSUE TITLE
Switching quality levels in a project where the URP Asset VolumeUpdateMode is set to ViaScripting throws exceptions

### ISSUE THREAD

**Issue by detzt** [detzt/BugReporting#1]
Can be fixed by adding this snippet to com.unity.render-pipelines.universal->UniversalRenderPipeline.cs line 1080:

```csharp

        // If an invalid volume stack is present, destroy it
        if (additionalCameraData.volumeStack != null && !additionalCameraData.volumeStack.isValid)
        {
            camera.DestroyVolumeStack(additionalCameraData);
        }

```

**Comment by detzt** [comment]
Fixed in Unity 2023.2.8 (URP Package 16.0.5)

**Comment by detzt** [comment]
Public issue https://issuetracker.unity3d.com/issues/nullreferenceexception-is-thrown-when-switching-between-quality-levels-with-post-processing-volume-update-mode-set-to-viascripting
